### PR TITLE
Wipe out unused code

### PR DESCRIPTION
**Pull request by malespiaut**
Here's already a big chunck of unused code removed.

### CODE REVIEW

**Comment by malespiaut** [comment]
I'm not sure why the Linux build fails, as this is not the case on my machine.

**Comment by malespiaut** [comment]
I see, an indentation warning. That goes as another argument for using clang-format at some point… Only saying. ;-)

**Comment by fabiangreffrath** [comment]
Incredible how much dead code this contains!

File: rott/cin_actr.c [hunk 1]
```diff
@@ -24,7 +24,6 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 #include "cin_efct.h"
 #include "modexlib.h"
 //MED
-#include "memcheck.h"
```
Comment (fabiangreffrath) [inline rott/cin_actr.c:27]: I guess the `//MED` comment wants to leave us together with the `#include "memcheck.h"` line.

Comment (malespiaut) [reply rott/cin_actr.c:27]: I've learned to ignore any comments in ROTT's source code. My bad!

**Pull request merged**